# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/wr/WebRenderCanvasRenderer.cpp
# Commit: 96da9d241051
# Full Hash: 96da9d241051d222bdd693bdb519d058a013b59c
# Author: Doug Thayer <dothayer@mozilla.com>
# Date: 2019-03-23 09:46:24
# Regressor Bug: 1441308
# File Overlap Count: 2
# Description:
#   Bug 1441308 - Core renderroot splitting changes r=kats,sotaro
#   
#   This is a large patch that contains all of the core changes for
#   renderroot splitting.
#   
# ==============================================================================

diff -r 8cefa694f811 -r 96da9d241051 gfx/layers/wr/WebRenderCanvasRenderer.cpp
--- a/gfx/layers/wr/WebRenderCanvasRenderer.cpp	Fri Mar 22 18:28:31 2019 +0000
+++ b/gfx/layers/wr/WebRenderCanvasRenderer.cpp	Fri Mar 22 18:28:42 2019 +0000
@@ -83,7 +83,7 @@
 
 void WebRenderCanvasRendererAsync::
     UpdateCompositableClientForEmptyTransaction() {
-  UpdateCompositableClient();
+  UpdateCompositableClient(mManager->GetRenderRoot());
   if (mPipelineId.isSome()) {
     // Notify an update of async image pipeline during empty transaction.
     // During non empty transaction, WebRenderBridgeParent receives